Deployments with deprecated runtimes (#3553)
astronomer/docs: astro/create-deployment.md

Every Deployment is hosted on an Astro cluster with its own dedicated resources that you can [customize](deployment-resources.md) to fine-tune your resource usage. To restrict communication between Deployments, resources for each Deployment are isolated within a corresponding Kubernetes namespace. See [Deployment network isolation](data-protection.md#deployment-network-isolation).

:::info

> If you're creating a Deployment to migrate to Astro Hosted from OSS Airflow or Astro Hybrid, and you need to use a deprecated version of the [Astro Runtime](runtime-version-lifecycle-policy.md), you can create Deployments with [deprecated versions of the Astro Runtime](upgrade-runtime.md#run-a-deprecated-astro-runtime-version).

Reviewer: ```suggestion
If you're migrating to Astro from OSS Airflow or another Astronomer product and you currently use an older version of Airflow, you can still create Deployments with the corresponding version of Astro Runtime even if it is deprecated according to the [Astro Runtime maintenance policy](runtime-version-lifecycle-policy.md#astro-runtime-maintenance-policy). This allows you to migrate your DAGs to Astro without needing to make any code changes and immediately upgrade to a new version of Airflow. Note that once you migrate your DAGs, Astronomer recommends upgrading to a supported version of Astro Runtime as soon as you can.

See [Run a deprecated Astro Runtime version](upgrade-runtime.md#run-a-deprecated-astro-runtime-version). 
```

Reviewer: @lzdanski I happened upon this PR as I was prepping for release preview and changed up wording! Sorry, feel free to take or leave

Every Deployment is hosted on an Astro cluster with its own dedicated resources that you can [customize](deployment-resources.md) to fine-tune your resource usage. To restrict communication between Deployments, resources for each Deployment are isolated within a corresponding Kubernetes namespace. See [Deployment network isolation](data-protection.md#deployment-network-isolation).

:::note

> If you're creating a Deployment to migrate to Astro Hosted from OSS Airflow or Astro Hybrid, and you need to use a deprecated version of the [Astro Runtime](runtime-version-lifecycle-policy.md), you can create Deployments with [deprecated versions of the Astro Runtime](upgrade-runtime.md#run-a-deprecated-astro-runtime).

Author: ```suggestion
If you're creating a Deployment to migrate to Astro Hosted from OSS Airflow or Astro Hybrid, and you need to use a deprecated version of the [Astro Runtime](runtime-version-lifecycle-policy.md), you can create Deployments with [deprecated versions of the Astro Runtime](upgrade-runtime.md#run-a-deprecated-astro-runtime-version).
```

> If you're creating a Deployment to migrate to Astro Hosted from OSS Airflow, Astro Hybrid, or Software, and you need to use a deprecated version of the [Astro Runtime](runtime-version-lifecycle-policy.md), you can create Deployments with [deprecated versions of the Astro Runtime](upgrade-runtime.md#run-a-deprecated-astro-runtime).

Reviewer 2: I don't think this applies to Software unfortunately.

Reviewer 2: Unless someone on the software team has also worked on this issue, @paolaperaza do you mind confirming here please?

Author: I think you're right! I misread Paola's first message. Making change.

> If you're creating a Deployment to migrate to Astro Hosted from OSS Airflow, Astro Hybrid, or Software, and you need to use a deprecated version of the [Astro Runtime](runtime-version-lifecycle-policy.md), you can create Deployments with [deprecated versions of the Astro Runtime](upgrade-runtime.md#run-a-deprecated-astro-runtime).

Author: ```suggestion
If you're creating a Deployment to migrate to Astro Hosted from OSS Airflow or Astro Hybrid, and you need to use a deprecated version of the [Astro Runtime](runtime-version-lifecycle-policy.md), you can create Deployments with [deprecated versions of the Astro Runtime](upgrade-runtime.md#run-a-deprecated-astro-runtime).
```
